csw-location/src/main/scala/csw/services/location/javadsl/ILocationService.scala

```diff
@@ -11,89 +11,82 @@ import csw.services.location.models._
 import csw.services.location.scaladsl.LocationService
 
 /**
-  * A `LocationService` interface which allows you to manage connections and their registrations.
-  * All operations are async, hence yield a `CompletableFuture`
+  * A LocationService interface to manage registrations. All operations are async, hence yield a CompletableFuture
   */
 trait ILocationService {
```

Comment:
There should be no mention of CRDT in interfaces! That is am implementation detail.